pkg/core/resources/apis/mesh/dataplane_validator.go

```diff
@@ -105,26 +98,6 @@ func validateNetworking(networking *mesh_proto.Dataplane_Networking) validators.
 	return err
 }
 
-func validateProbes(probes *mesh_proto.Dataplane_Probes) validators.ValidationError {
-	if probes == nil {
-		return validators.ValidationError{}
-	}
-	var err validators.ValidationError
-	path := validators.RootedAt("probes")
-	err.Add(ValidatePort(path.Field("port"), probes.GetPort()))
-	for i, endpoint := range probes.Endpoints {
-		indexPath := path.Field("endpoints").Index(i)
-		err.Add(ValidatePort(indexPath.Field("inboundPort"), endpoint.GetInboundPort()))
-		if _, URIErr := url.ParseRequestURI(endpoint.InboundPath); URIErr != nil {
-			err.AddViolationAt(indexPath.Field("inboundPath"), `should be a valid URL Path`)
-		}
-		if _, URIErr := url.ParseRequestURI(endpoint.Path); URIErr != nil {
-			err.AddViolationAt(indexPath.Field("path"), `should be a valid URL Path`)
-		}
-	}
-	return err
-}
```

Comment:
Why have we lost validation? It was optional to start with right?

Reply:
Ah, thanks for reminding. I'll bring it back.
I removed this feature on the `Dataplane` object so this validation was removed as well. But after the first round of review, I restored that field but forgot to restore this validation.